js/lang/es.js

```diff
@@ -8,6 +8,7 @@ app.lang['es'] = (function() {
 		'language.cs': 'Checo',
 		'language.en': 'Inglés',
 		'language.es': 'Español',
+		'language.fr': 'Francais',
```

Comment:
Is that Spanish? I thought it was "Francés".

Reply:
frances is not french